tensorrt_llm/_torch/pyexecutor/_util.py

```
@@ -361,71 +374,74 @@ def configure_kv_cache_capacity(self, py_executor: PyExecutor) -> None:
-        )
-
-        # get kv cache stats for both model and draft model
-        kv_stats = py_executor.resource_manager.resource_managers.get(
-            ResourceManagerType.KV_CACHE_MANAGER).get_kv_cache_stats()
-        # Get draft KV cache stats if present (either from two-model mode or one-model
-        # mode with separate draft KV cache)
-        draft_kv_cache_manager = py_executor.resource_manager.resource_managers.get(
-            ResourceManagerType.DRAFT_KV_CACHE_MANAGER)
-        kv_stats_draft = draft_kv_cache_manager.get_kv_cache_stats(
-        ) if draft_kv_cache_manager is not None else None
+        if py_executor is not None and self._skip_est == False:
```

Ruff (0.15.2)

[error] Avoid equality comparisons to `False`; use `not self._skip_est:` for false checks

Replace with `not self._skip_est`

(E712)